### ISSUE TITLE
List currently active rules

### ISSUE THREAD

**Issue by haluzpav** [detekt/detekt#5524]
When I run `./gradlew detekt`, it lists only issues in my code.

I'd like to see a list of currently active **rules**, combining the default and my own configs. This would be an useful overview, since I disable many rules in my config, and it's hard to track which are actually enabled.

Bonus points for comparison with the default config, so I see which custom configs I could delete.

I'd like a table such as this in a report for example.

|ruleset| rule | default config | custom config |
|-| ---- | - | - |
|formatting|AnnotationOnSeparateLine|**disabled**|-|
|formatting|ChainWrapping|enabled|**disabled**|
|...||||

**Comment by chao2zhang** [comment]
Hello @haluzpav, 
- 1 This seems like a useful detekt cli parameter to print the active rules. 
- 2 The diff comparison could be something customized in your gradle configuration.

Please let us know if you are willing to contribute to task 1.

**Comment by haluzpav** [comment]
@chao2zhang Thanks for approval. Maybe I could do it.

It basically comes down to adding something like this into [`Analyzer`](https://github.com/detekt/detekt/blob/main/detekt-core/src/main/kotlin/io/gitlab/arturbosch/detekt/core/Analyzer.kt) and making it prettier (return instead of `println` etc.), right? 

```kotlin
activeRuleSetsToRuleSetConfigs.forEach { (ruleSet, _) ->
    ruleSet.rules
        .flatMap { rule ->
            when (rule) {
                is ConfigAware -> listOf(rule)
                is MultiRule -> rule.rules
                else -> error("Unknown rule type $rule")
            }
        }
        .filter { it.active }
        .forEach { println("${it.ruleId} is active") }
}
```

**Comment by cortinico** [comment]
We do have work pending on this here:
- https://github.com/detekt/detekt/pull/5885

**Comment by cortinico** [comment]
Reopening as this is still up for grab

**Comment by schalkms** [comment]
Quite similar to issue #4925
Might make sense to merge those two ideas.
